metric_learn/lmnn.py

```diff
@@ -102,6 +156,8 @@ def fit(self, X, y):
         self._loss_grad(X, L, dfG, impostors, 1, k, reg, target_neighbors, df,
                         a1, a2))
 
+    it = 1  # we already made one iteration
```

Comment:
This seems like a no-op line. Maybe just update the "main loop" comment?

Reply:
I am not sure I understand, I think the `it=1` still useful for coherence, if one puts `max_iter=1`, it would break otherwise (variable not defined)

Reply:
Ah, I didn't see that we're referencing `it` after the loop.